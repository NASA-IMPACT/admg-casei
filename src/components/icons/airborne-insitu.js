import React from "react"
import PropTypes from "prop-types"
import { sizes } from "./utils"

export const AirborneInsitu = ({ color = "#FFF", size = "large" }) => (
  <svg
    viewBox="0 0 80 80"
    width={sizes[size].width}
    height={sizes[size].height}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      fill={color}
      fillRule="evenodd"
      d="M49.337 76.811l.29 2.233c-1.487.193-3 .29-4.492.29-1.478 0-2.959-.095-4.408-.281l.288-2.233a32.7 32.7 0 008.322-.009zm-20.1-3.956c2.414 1.386 5 2.453 7.689 3.166l-.578 2.176a34.101 34.101 0 01-8.232-3.39l1.12-1.952zm31.867-.043l1.127 1.947a33.955 33.955 0 01-8.227 3.414l-.583-2.176a31.644 31.644 0 007.683-3.185zm-41.24-8.13a32.15 32.15 0 005.905 5.862l-1.368 1.79a34.426 34.426 0 01-6.315-6.273l1.778-1.38zm50.592-.063l1.783 1.37a34.24 34.24 0 01-6.301 6.29l-1.369-1.784a32.196 32.196 0 005.887-5.876zm-25.32-42.593c12.74 0 23.106 10.366 23.106 23.107 0 12.742-10.365 23.107-23.107 23.107-12.741 0-23.109-10.365-23.109-23.107 0-12.741 10.368-23.107 23.11-23.107zm-9.17 33.866l-.321.089c1.802 5.382 4.875 9.212 8.365 9.898V54.743c-2.811.084-5.547.482-8.044 1.149zm10.295-1.149V65.88c3.489-.686 6.562-4.516 8.364-9.897-2.582-.721-5.432-1.15-8.364-1.239zM33.45 56.686l-.202.074c-1.45.546-2.781 1.193-3.952 1.929a20.933 20.933 0 008.59 6.002c-1.838-1.995-3.362-4.74-4.436-8.005zm23.372 0l-.105.308c-1.068 3.13-2.553 5.765-4.334 7.696a20.932 20.932 0 008.591-6.001c-1.224-.77-2.624-1.441-4.152-2.003zm-42.514-3.123a31.941 31.941 0 003.222 7.67l-1.944 1.133a34.058 34.058 0 01-3.448-8.211l2.17-.592zm61.673-.077l2.174.585a33.939 33.939 0 01-3.43 8.219l-1.944-1.13a31.792 31.792 0 003.2-7.674zM31.722 46.26h-7.415a20.729 20.729 0 003.612 10.637c1.448-.937 3.098-1.738 4.9-2.391-.635-2.53-1.018-5.308-1.097-8.246zm34.24 0h-7.414c-.078 2.938-.461 5.716-1.097 8.246 1.802.653 3.453 1.454 4.901 2.39a20.736 20.736 0 003.61-10.636zm-9.666 0l-10.036-.001v6.233c3.177.087 6.228.539 9.003 1.314.59-2.33.953-4.88 1.033-7.546zm-12.286-.001H33.975c.08 2.666.443 5.216 1.033 7.547 2.775-.775 5.826-1.227 9.002-1.314v-6.233zm-30.825-1.125c0 1.452.099 2.912.292 4.341l-2.23.304a34.756 34.756 0 01-.313-4.645h2.25zm65.877-4.332a34.66 34.66 0 01-.03 8.898l-2.232-.3a32.331 32.331 0 00.03-8.317l2.232-.28zm-51.143-7.43l-.098.143a20.726 20.726 0 00-3.514 10.493h7.415c.079-2.938.462-5.718 1.098-8.248-1.802-.652-3.452-1.453-4.901-2.388zm7.09 3.09l-.033.127a35.3 35.3 0 00-1.001 7.42l10.035-.001v-6.232c-3.176-.087-6.227-.54-9.001-1.315zm20.253 0l-.013.003c-2.77.773-5.818 1.224-8.989 1.31v6.233h10.036a35.235 35.235 0 00-1.034-7.547zm7.088-3.091l-.139.09c-1.415.898-3.016 1.669-4.76 2.3.636 2.53 1.019 5.309 1.097 8.247h7.414A20.73 20.73 0 0062.35 33.37zM32.11 0l5.442 5.441-9.53 9.528-1.926-1.926-2.478 2.481a5.78 5.78 0 01.99 3.25c0 1.558-.608 3.02-1.71 4.123a5.79 5.79 0 01-4.123 1.708 5.789 5.789 0 01-3.25-.988l-2.479 2.476 1.926 1.926-9.53 9.53L0 32.108l9.53-9.53 1.925 1.924 2.479-2.477a5.85 5.85 0 010-6.501l-1.706-1.707 1.591-1.59 1.706 1.706a5.85 5.85 0 016.5 0l2.48-2.481-1.925-1.924L32.11 0zm42.73 28.179a33.932 33.932 0 013.373 8.24l-2.176.572a31.768 31.768 0 00-3.15-7.696l1.953-1.116zm-39.191 6.094l-.005.013c2.583.72 5.433 1.15 8.367 1.237V24.387c-3.489.685-6.56 4.51-8.362 9.886zm10.612-9.886v11.136c2.932-.088 5.782-.518 8.365-1.237-1.803-5.383-4.876-9.213-8.365-9.899zM9.53 25.76l-6.349 6.348 2.26 2.26 6.348-6.349-2.26-2.26zm28.357-.186l-.024.01a20.943 20.943 0 00-8.568 5.993c1.225.77 2.625 1.443 4.154 2.004 1.075-3.266 2.6-6.012 4.438-8.007zm14.494 0l.028.032c1.826 1.991 3.34 4.725 4.41 7.973 1.529-.56 2.929-1.232 4.153-2.001a20.932 20.932 0 00-8.591-6.003zm-20.517-6.8c0 7.22-5.871 13.091-13.09 13.091v-2.25c5.979 0 10.84-4.863 10.84-10.84h2.25zm-3.504 0c0 5.285-4.3 9.585-9.586 9.585v-2.25c4.046 0 7.336-3.29 7.336-7.334h2.25zm37.759-.65a34.52 34.52 0 016.258 6.333l-1.791 1.361a32.137 32.137 0 00-5.849-5.916l1.382-1.778zm-47.345-2.927c-.916 0-1.834.346-2.532 1.046a3.586 3.586 0 000 5.063c1.355 1.353 3.71 1.353 5.063 0a3.56 3.56 0 001.051-2.531c0-.957-.373-1.857-1.05-2.532a3.558 3.558 0 00-2.532-1.046zm35.454-3.043a34.061 34.061 0 018.2 3.47l-1.14 1.938a31.841 31.841 0 00-7.659-3.236l.599-2.172zm-9.094-1.222c1.553 0 3.115.106 4.645.315l-.304 2.228a32.46 32.46 0 00-4.34-.292v-2.25zM32.11 3.182l-6.348 6.346 2.26 2.26L34.37 5.44l-2.26-2.259z"
    />
  </svg>
)

AirborneInsitu.propTypes = {
  color: PropTypes.string,
  size: PropTypes.string,
}
