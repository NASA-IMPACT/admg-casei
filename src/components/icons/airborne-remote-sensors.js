import React from "react"
import PropTypes from "prop-types"
import { sizes } from "./utils"

export const AirborneRemoteSensors = ({ color = "#FFF", size = "large" }) => (
  <svg
    viewBox="0 0 80 120"
    width={sizes[size].width}
    height={sizes[size].height}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      fill={color}
      fillRule="evenodd"
      d="M7.378 70.315l1.767 1.767-4.879 4.877 3.301.001v2.5H0v-7.567h2.5v3.298l4.878-4.876zM61.917 3.45L76.01 17.543 58.835 34.718l-6.163-6.162-2.308 2.308 8.799 8.798c3.203-.678 5.87-.124 7.605 1.613l.885.885L55.79 54.022l1.107 1.108a3.986 3.986 0 014.593.74 3.982 3.982 0 010 5.62 3.957 3.957 0 01-2.81 1.163 3.962 3.962 0 01-2.81-1.163 3.985 3.985 0 01-.74-4.593l-1.107-1.107-11.866 11.865-.882-.885c-1.775-1.775-2.26-4.498-1.613-7.606l-8.801-8.801-2.308 2.308 6.165 6.164L17.542 76.01 3.45 61.917l17.175-17.175 6.161 6.162 2.308-2.308-7.279-7.278L26.35 26.35l14.965-4.535 7.28 7.28 2.308-2.308-6.163-6.162L61.917 3.45zM20.624 48.278l-13.64 13.64 10.558 10.557 13.64-13.64-4.397-4.396L15.037 66.19l-1.767-1.767L25.019 52.67l-4.394-4.393zm49.61 14.392v2.5l-3.298-.001 4.876 4.876-1.768 1.767-4.878-4.877v3.3h-2.5V62.67h7.568zm-8.572-20.78c-.978 0-2.078.185-3.26.555-3.223 1.01-6.666 3.235-9.693 6.265-5.33 5.332-7.97 11.875-6.392 15.25L63.95 42.327c-.645-.292-1.415-.437-2.288-.437zM58.68 57.208c-.377 0-.755.145-1.042.43a1.48 1.48 0 000 2.084c.577.578 1.51.578 2.084 0a1.476 1.476 0 000-2.084 1.479 1.479 0 00-1.042-.43zM40.608 24.642l-12.253 3.713L24.64 40.61l15.84 15.838c.278-.719.61-1.448.991-2.183l-8.753-8.755 1.767-1.77 8.292 8.296a31.898 31.898 0 014.166-5.093c2.94-2.94 6.273-5.202 9.506-6.46l-15.841-15.84zM30.684 39.94l2.31 2.313-1.767 1.767-2.31-2.31 1.767-1.77zM61.917 6.985l-13.642 13.64 4.395 4.394L64.42 13.27l1.77 1.768-11.75 11.75 4.395 4.395 13.64-13.64L61.918 6.985zM79.46 0v7.567h-2.5V4.265l-4.877 4.88-1.768-1.767L75.19 2.5h-3.297V0h7.567z"
    />
  </svg>
)

AirborneRemoteSensors.propTypes = {
  color: PropTypes.string,
  size: PropTypes.string,
}
