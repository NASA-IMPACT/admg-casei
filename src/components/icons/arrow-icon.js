import React from "react"
import PropTypes from "prop-types"

const ArrowIcon = ({ color = "#FFF" }) => (
  <svg
    style={{ marginLeft: `.5rem` }}
    version="1.1"
    xmlns="http://www.w3.org/2000/svg"
    width="16"
    height="16"
    viewBox="0 0 16 16"
    role="img"
  >
    <title>Arrow Right</title>
    <rect width="16" height="16" fill="none" />
    <polygon
      fill={color}
      points="7.586,2.414 12.172,7 0,7 0,9 12.172,9 7.586,13.586 9,15 16,8 9,1"
    />
  </svg>
)

ArrowIcon.propTypes = {
  color: PropTypes.string,
}

export default ArrowIcon
