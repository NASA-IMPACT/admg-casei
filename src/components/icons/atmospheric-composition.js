import React from "react"
import PropTypes from "prop-types"

export const AtmosphericComposition = ({ color = "#FFF" }) => (
  <svg width="120" height="120" xmlns="http://www.w3.org/2000/svg">
    <path
      fill={color}
      fillRule="nonzero"
      opacity=".9"
      d="M119.851 55.815c2.312 33.055-22.611 61.725-55.666 64.036-24.75 1.731-47.041-11.806-57.651-32.593a2.764 2.764 0 01-.283-.565A59.798 59.798 0 01.148 64.185c-.882-12.618 2.193-24.92 8.646-35.461l.86-1.365 1.974 1.281C4.865 39.053 1.61 51.368 2.495 64.021a57.558 57.558 0 004.527 18.74 1.175 1.175 0 011.974 1.171c-.2.614-.39 1.231-.568 1.852a57.734 57.734 0 0015.27 18.995 38.561 38.561 0 01-.167-3.604c0-21.224 17.205-38.43 38.43-38.43 21.224 0 38.43 17.206 38.43 38.43v-.046c11.608-11.41 18.337-27.64 17.113-45.15C115.284 24.22 87.737.275 55.979 2.496c-8.879.62-17.324 3.234-24.845 7.592l-1.492.895-1.24-2C36.612 3.889 45.96.839 55.814.15c33.055-2.312 61.725 22.61 64.036 55.666zM76.76 68.263l-2.802.869c-.363.09-.819.208-1.346.356-1.466.41-2.931.892-4.292 1.439-3.695 1.485-5.8 3.152-5.675 4.553.22 2.523 1.881 3.361 4.158 2.158 1.524-.8 2.724.06 3.2 1.67.288.97.39 2.309.36 3.961a39.144 39.144 0 01-.098 2.07l-.088 1.066a43.297 43.297 0 01-.19 1.721 1.176 1.176 0 01-1.166 1.018c-.479 0-.861.052-1.915.294l-2.745.647c-.362.079-.69.144-1.023.202-2.794.49-4.917.258-6.513-1.107l-.422-.4-.325-.812-.146-.181c-.63-.665-2.392-.93-5.783-.212-2.545.534-4.274 2.426-5.342 5.35-.468 1.28-.772 2.666-.941 4.055a16.323 16.323 0 00-.135 1.82c0 .652.123 1.692.483 2.703.539 1.513 1.395 2.315 2.75 2.315l.704-.011 2.816-.144c1.875-.069 2.787.047 3.822.529 1.638.763 2.507 2.326 2.64 4.763l.004.212.24.27.658.787.347.445c.286.375.544.74.768 1.094.6.948.944 1.762.944 2.52 0 .761-.163 1.324-.607 2.303l-.456.973-.033.073a58.36 58.36 0 0013.853-1.343c.472-.37.937-.791 1.386-1.264.432-.454.526-.56.135-.963l-.682-.64c-.846-.787-1.244-1.351-1.244-2.237 0-.718.385-1.287 1.34-2.381l2.31-2.568c.994-1.14 1.726-2.099 2.25-2.992.69-1.174.941-2.086.765-2.807l-.061-.176-.521.167a6.626 6.626 0 01-1.1.21l-.547.029c-.86-.002-1.642-.227-2.28-.725l-.366-.332-.291-.539a30.13 30.13 0 00-.882-2.972c-.476-1.34-1.054-2.313-.912-2.313-.87 0-1.61-.69-1.36-1.666.103-.398.348-.722.717-1.069.573-.54 1.483-1.13 2.527-1.63 2.456-1.175 5.21-1.694 7.84-1.233l.37.081.328-.188c1.318-.852 2.453-2.075 3.413-3.597l.465-.785c.73-1.31 1.3-2.721 1.725-4.134l.255-.914c.07-.27.3-1.241.457-1.916a36.188 36.188 0 00-11.741-8.472zM61.96 65.098c-19.924 0-36.077 16.152-36.077 36.077 0 1.907.149 3.78.434 5.608 8.54 6.154 18.793 10.01 29.807 10.738.044-.136.096-.275.154-.42l.783-1.726c.216-.507.292-.802.292-1.092 0-.188-.2-.663-.579-1.26a13.402 13.402 0 00-.652-.93l-.616-.77c-.197-.237-.384-.45-.546-.629l-.225-.242-.325-.81c0-2-.45-2.922-1.298-3.317-.683-.319-1.48-.378-3.407-.28l-1.754.097c-.449.02-.8.03-1.188.03-2.503 0-4.127-1.522-4.967-3.88a10.998 10.998 0 01-.542-2.386l-.06-.646a7.296 7.296 0 01-.017-.46l.02-.66c.023-.396.064-.884.132-1.445.19-1.555.53-3.112 1.067-4.578 1.32-3.615 3.608-6.118 7.066-6.845 4.108-.868 6.668-.484 7.976.896.35.37.566.754.682 1.117l.021.086.07.064c.858.68 2.09.866 3.84.639l.68-.105 1.15-.232 2.92-.688.963-.187.07-.675a40.44 40.44 0 00.175-2.96c.021-1.192-.035-2.181-.171-2.873l-.091-.378a.654.654 0 01-.032-.166l-.301.146c-3.543 1.576-6.761-.217-7.114-4.27-.258-2.892 2.5-5.077 7.142-6.943a44.237 44.237 0 014.535-1.52l1.258-.329a36.045 36.045 0 00-11.275-1.796zm28.43 13.864l-.007.031c-.087.358-.2.779-.342 1.25a23.24 23.24 0 01-1.922 4.6c-1.252 2.247-2.825 4.063-4.768 5.285l-.745.43-.848.096c-2.285-.602-4.831-.193-7.096.89-.336.16-.65.33-.932.5l-.343.213.139.166c.41.544.773 1.308 1.143 2.28l.223.607.362 1.088.416 1.44.078.302.192.093.256.06.302.02c.421.001.911-.102 1.41-.278l.504-.2.154-.075a1.176 1.176 0 011.403.23c1.653 1.742 1.473 3.967.018 6.445-.489.834-1.107 1.682-1.888 2.622l-1.076 1.241c-.663.738-1.66 1.831-1.852 2.052l-.612.734c-.065.075-.11.12-.131.125l.28.301.41.385c1.243 1.143 1.815 2.179 1.381 3.35a57.373 57.373 0 0021.471-11.869 35.82 35.82 0 00.067-2.201 35.922 35.922 0 00-7.647-22.213zm18.771-9.529a1.176 1.176 0 011.626.352 57.795 57.795 0 013.86 7.015 1.176 1.176 0 11-2.136.988 55.443 55.443 0 00-3.702-6.728 1.176 1.176 0 01.352-1.627zm-95.539-.387a1.176 1.176 0 111.959 1.304 55.74 55.74 0 00-3.806 6.67 1.176 1.176 0 01-2.12-1.022 58.092 58.092 0 013.967-6.952zm83.665-39.32a2.615 2.615 0 01-.722 3.42L97.19 40a3.921 3.921 0 11-2.612 7.321l-2.457 2.226a3.922 3.922 0 01-1.518 5.169l1.223 6.246a2.614 2.614 0 11-2.308.444l-1.207-6.169a3.917 3.917 0 01-3.573-3.401l-4.078-.678a2.614 2.614 0 11.413-2.317l4.075.679a3.919 3.919 0 015.325-1.652l2.434-2.207-.023-.042a3.922 3.922 0 011.963-5.394l-.612-6.699a2.614 2.614 0 113.052-3.8zm1.41 28.034a1.176 1.176 0 011.661-.104 58.364 58.364 0 015.624 5.695 1.176 1.176 0 01-1.783 1.535A56.01 56.01 0 0098.8 59.42a1.176 1.176 0 01-.103-1.66zm-74.252-.873a1.176 1.176 0 111.522 1.794 56.005 56.005 0 00-5.503 5.36 1.176 1.176 0 01-1.754-1.569 58.357 58.357 0 015.735-5.585zm13.509-8.57a1.176 1.176 0 01.973 2.142 55.427 55.427 0 00-6.753 3.654 1.176 1.176 0 01-1.26-1.987 57.777 57.777 0 017.04-3.809zm49.89 1.653a1.569 1.569 0 101.569 2.717 1.569 1.569 0 00-1.569-2.717zm-34.459-6.2a1.176 1.176 0 01.345 2.327 55.536 55.536 0 00-7.504 1.648 1.176 1.176 0 01-.662-2.258 57.889 57.889 0 017.821-1.717zM18.825 9.022a5.882 5.882 0 012.487 11.214l1.083 4.482a5.882 5.882 0 016.183 6.642l8.614 2.846a3.921 3.921 0 11-.72 2.266v-.031l-8.66-2.861a5.88 5.88 0 01-5.066 2.892c-.961 0-1.87-.231-2.671-.64l-3.995 4.79a3.921 3.921 0 11-1.917-1.374l4.072-4.884a5.882 5.882 0 011.887-9.04l-1.097-4.541a5.882 5.882 0 11-.2-11.76zm42.541 34.117a58.2 58.2 0 017.991.461 1.176 1.176 0 11-.297 2.334c-2.53-.322-5.09-.47-7.668-.442a1.176 1.176 0 11-.026-2.353zm34.227-.661a1.569 1.569 0 101.569 2.716 1.569 1.569 0 00-1.569-2.716zm-82.65-1.071a1.569 1.569 0 100 3.137 1.569 1.569 0 000-3.137zm27.45-6.504a1.569 1.569 0 100 3.137 1.569 1.569 0 000-3.137zM22.746 27.06a3.53 3.53 0 100 7.059 3.53 3.53 0 000-7.059zm-3.921-15.685a3.53 3.53 0 100 7.058 3.53 3.53 0 000-7.058z"
    />
  </svg>
)

AtmosphericComposition.propTypes = {
  color: PropTypes.string,
}
