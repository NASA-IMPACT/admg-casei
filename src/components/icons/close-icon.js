import React from "react"
import PropTypes from "prop-types"

const CloseIcon = ({ color = "#FFF" }) => (
  <svg
    xmlns="http://www.w3.org/2000/svg"
    width="12px"
    height="12px"
    viewBox="0 0 16 16"
  >
    <polygon
      fill={color}
      points="13.707,3.707 12.293,2.293 8,6.586 3.707,2.293 2.293,3.707 6.586,8 2.293,12.293 3.707,13.707 8,9.414  12.293,13.707 13.707,12.293 9.414,8"
    />
  </svg>
)

CloseIcon.propTypes = {
  color: PropTypes.string,
}

export default CloseIcon
