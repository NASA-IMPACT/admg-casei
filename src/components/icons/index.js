import CloseIcon from "./close-icon"
import TrashIcon from "./trash-icon"
import ArrowIcon from "./arrow-icon"
import ChevronIcon from "./chevron-icon"

export { CloseIcon, TrashIcon, ArrowIcon, ChevronIcon }

export { AtmosphericComposition as AtmosphericCompositionIcon } from "./atmospheric-composition"
export { AtmosphericDynamics as AtmosphericDynamicsIcon } from "./atmospheric-dynamics"
export { CarbonCycleEcosystems as CarbonCycleEcosystemsIcon } from "./carbon-cycle_ecosystems"
export { ClimateVariabilityChange as ClimateVariabilityChangeIcon } from "./climate-variability_change"
export { EarthSurfaceInterior as EarthSurfaceInteriorIcon } from "./earth-surface_interior"
export { GlobalWaterEnergyCycle as GlobalWaterEnergyCycleIcon } from "./global-water_energy-cycle"
export { Weather as WeatherIcon } from "./weather"

export { AirborneRemoteSensors } from "./airborne-remote-sensors"
