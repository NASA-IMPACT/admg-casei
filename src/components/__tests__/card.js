import React from "react"
import renderer from "react-test-renderer"
import Card from "../cards/card"
import { CampaignIcon } from "../../icons"

const CardContent = () => (
  <div>
    <big
      style={{ fontWeight: `bold`, marginTop: `0.5rem` }}
      data-cy="shortname"
    >
      GCPEx
    </big>
    <p data-cy="longname">GPM Cold Season Precipitation Experiment</p>
    <p data-cy="description">description....</p>
  </div>
)
const testTag = "Stationary"
const testFooterList = [
  { count: 1, title: "Collection Periods" },
  { count: 0, title: "Data Products" },
]

describe("Card", () => {
  it("renders plain card", () => {
    const tree = renderer
      .create(
        <Card placeholder={CampaignIcon} category="campaigns">
          <CardContent />
        </Card>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders card with tag", () => {
    const tree = renderer
      .create(
        <Card placeholder={CampaignIcon} category="campaigns" tag={testTag}>
          <CardContent />
        </Card>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders with footer list", () => {
    const tree = renderer
      .create(
        <Card
          placeholder={CampaignIcon}
          category="campaigns"
          testFooterList={testFooterList}
        >
          <CardContent />
        </Card>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
