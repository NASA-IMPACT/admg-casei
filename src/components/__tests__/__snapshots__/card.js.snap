// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders card with tag 1`] = `
.c0.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0.5rem 0 2rem 0;
  min-height: 2rem;
}

.c0.c0 div:only-child {
  margin-left: auto;
}

<div
  data-cy="campaigns-card"
  style={
    Object {
      "backgroundColor": "#303641",
      "boxShadow": "rgba(68, 63, 63, 0.08) 0px -1px 1px 0px, rgba(68, 63, 63, 0.08) 0px 2px 6px 0px",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "space-between",
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "marginBottom": "2rem",
      }
    }
  >
    <div
      className="c0"
    >
      <img
        alt="placeholder logo"
        height="85"
        src="test-file-stub"
      />
      <div
        data-cy="stationary-tag"
        style={
          Object {
            "border": "1px solid",
            "padding": "0.25rem",
            "textTransform": "uppercase",
          }
        }
      >
        Stationary
      </div>
    </div>
    <div>
      <big
        data-cy="shortname"
        style={
          Object {
            "fontWeight": "bold",
            "marginTop": "0.5rem",
          }
        }
      >
        GCPEx
      </big>
      <p
        data-cy="longname"
      >
        GPM Cold Season Precipitation Experiment
      </p>
      <p
        data-cy="description"
      >
        description....
      </p>
    </div>
  </div>
</div>
`;

exports[`Card renders plain card 1`] = `
.c0.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0.5rem 0 2rem 0;
  min-height: 2rem;
}

.c0.c0 div:only-child {
  margin-left: auto;
}

<div
  data-cy="campaigns-card"
  style={
    Object {
      "backgroundColor": "#303641",
      "boxShadow": "rgba(68, 63, 63, 0.08) 0px -1px 1px 0px, rgba(68, 63, 63, 0.08) 0px 2px 6px 0px",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "space-between",
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "marginBottom": "2rem",
      }
    }
  >
    <div
      className="c0"
    >
      <img
        alt="placeholder logo"
        height="85"
        src="test-file-stub"
      />
    </div>
    <div>
      <big
        data-cy="shortname"
        style={
          Object {
            "fontWeight": "bold",
            "marginTop": "0.5rem",
          }
        }
      >
        GCPEx
      </big>
      <p
        data-cy="longname"
      >
        GPM Cold Season Precipitation Experiment
      </p>
      <p
        data-cy="description"
      >
        description....
      </p>
    </div>
  </div>
</div>
`;

exports[`Card renders with footer list 1`] = `
.c0.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0.5rem 0 2rem 0;
  min-height: 2rem;
}

.c0.c0 div:only-child {
  margin-left: auto;
}

<div
  data-cy="campaigns-card"
  style={
    Object {
      "backgroundColor": "#303641",
      "boxShadow": "rgba(68, 63, 63, 0.08) 0px -1px 1px 0px, rgba(68, 63, 63, 0.08) 0px 2px 6px 0px",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "space-between",
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "marginBottom": "2rem",
      }
    }
  >
    <div
      className="c0"
    >
      <img
        alt="placeholder logo"
        height="85"
        src="test-file-stub"
      />
    </div>
    <div>
      <big
        data-cy="shortname"
        style={
          Object {
            "fontWeight": "bold",
            "marginTop": "0.5rem",
          }
        }
      >
        GCPEx
      </big>
      <p
        data-cy="longname"
      >
        GPM Cold Season Precipitation Experiment
      </p>
      <p
        data-cy="description"
      >
        description....
      </p>
    </div>
  </div>
</div>
`;
